pkgs/development/compilers/fpc/default.nix

```diff
@@ -17,7 +17,14 @@ stdenv.mkDerivation rec {
   # Patch paths for linux systems. Other platforms will need their own patches.
   patches = [
     ./mark-paths.patch # mark paths for later substitution in postPatch
-  ];
+  ] ++ lib.optional stdenv.isAarch64 (fetchpatch {
+    # backport upstream patch for aarch64 glibc 2.34
+    url = "https://gitlab.com/freepascal.org/fpc/source/-/commit/a20a7e3497bccf3415bf47ccc55f133eb9d6d6a0.patch";
```

Comment:
https://gitlab.com/freepascal.org/fpc/source/-/commit/a20a7e3497bccf3415bf47ccc55f133eb9d6d6a0

Reply:
??

Reply:
I was just double checking if the commit was already merged on master.